src/xdocs/checks.xml

```diff
@@ -805,5 +799,26 @@
       </tr>
     </table>
     </section>
+
+    <section name="Non-Standard Checks">
```

Comment:
is it better to name them "Non-Checks" or "Holders" ?
I used "non-standard checks" for Checks from extensions.